docs/usage/quickstart.rst

```diff
@@ -3,112 +3,154 @@
+
+        [kube-master]
+        node-01
+        node-02
+        node-03
+
+        [etcd]
+        node-01
+        node-02
+        node-03
+
+        [kube-node]
+        node-01
+        node-02
+        node-03
 
-.. code-block:: ini
+        [k8s-cluster:children]
+        kube-node
+        kube-master
 
-    node-01 ansible_host=10.0.0.1 ansible_user=centos
-    node-02 ansible_host=10.0.0.2 ansible_user=centos
-    node-03 ansible_host=10.0.0.3 ansible_user=centos
+    Change the IP addresses, domain names, and user names to conform to
+    your infrastructure.
 
-    [kube-master]
-    node-01
-    node-02
-    node-03
+ 3. Create a :file:`group_vars` subdirectory in the inventory directory.
 
-    [etcd]
-    node-01
-    node-02
-    node-03
+  - A second file, called :file:`kube-node.yml`, in a :file:`group_vars`
+    subdirectory of the inventory, declares how to set up storage (in the
+    default configuration) on hosts in the *kube-node* group; that is,
+    hosts on which Pods will be scheduled
 
-    [kube-node]
-    node-01
-    node-02
-    node-03
+     .. code:: shell
 
-    [k8s-cluster:children]
-    kube-node
-    kube-master
+      $ cd ..
+      $ mkdir group_vars ; cd group_vars
 
-Make sure to change IP-addresses, usernames etc. according to your
-infrastructure.
+  - In group_vars, create a file named :file:`kube-node.yml`. This file
+    declares the default storage configuration on hosts in the *kube-node*
+    group—that is, hosts on which Pods shall be scheduled
 
-In a second file, called :file:`kube-node.yml` in a :file:`group_vars`
-subdirectory of our inventory, we declare how to setup storage (in the
-default configuration) on hosts in the *kube-node* group, i.e. hosts on which
-Pods will be scheduled:
+     .. code-block:: yaml
 
-.. code-block:: yaml
+         metal_k8s_lvm:
+           vgs:
+             kubevg:
+               drives: ['/dev/vdb']
 
-    metal_k8s_lvm:
-      vgs:
-        kubevg:
-          drives: ['/dev/vdb']
+    In this example, every *kube-node* host is assumed to have a disk available
+    as :file:`/dev/vdb` which can be used to set up Kubernetes
+    *PersistentVolumes*. For more information about storage, see
+    :doc:`../architecture/storage`.
 
-In the above, we assume every *kube-node* host has a disk available as
-:file:`/dev/vdb` which can be used to set up Kubernetes *PersistentVolumes*. For
-more information about storage, see :doc:`../architecture/storage`.
+Clone or Copy the MetalK8s Git Repo
+-----------------------------------
 
-Entering the MetalK8s Shell
----------------------------
-To easily install a supported version of Ansible and its dependencies, as well
-as some Kubernetes tools (:program:`kubectl` and :program:`helm`), we provide a
-:program:`make` target which installs these in a local environment. To enter this
-environment, run :command:`make shell` (this takes a couple of seconds on first
-run)::
+Return to the home subdirectory.
```

Comment:
No no no, let's not tell the user to clone stuff in a home-dir. If we want to add these instructions (I don't think we should, by the way), they should be at the very beginning.

Reply:
I agree, it's none of our business to tell people where they want to keep their scripts.